src/services/swap_service.ts

```diff
@@ -51,6 +52,7 @@ export class SwapService {
             rfqt: {
                 takerApiKeyWhitelist: RFQT_API_KEY_WHITELIST,
                 makerEndpoints: RFQT_MAKER_ENDPOINTS,
+                warningLogger: logger.warn.bind(logger),
```

Comment:
Is this bind meaningful? I don't think it is -- if this was only done because of a tslint issue, perhaps we can just tslint:disable-next-line. If we need to keep the .bind I think it should be .bind(this) 🤔

Reply:
`tslint` doesn't complain if I remove the `.bind()`, but it blows up at runtime.  and if i change it to `.bind(this)` then it blows up in the same way.  see details at https://gist.github.com/feuGeneA/17fef8fd4418c72ba712c62826bdfafa

I believe what's happening is that the `logger.warn()` method has references to `this`, and without the `.bind()` those expected references aren't defined.  I believe using `.bind(this)` would cause `logger.warn()` to resolve those references by looking for them on the `SwapQuoter` object that we're instantiating here, rather than on the `logger` object, which would obviously not be what we want.  I do believe this `.bind` is meaningful.